src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPaginatedItems(items: string[], perPage: number) {
+  const paginated = [];
+
+  for (let i = 1; i < Math.ceil(items.length / perPage) + 1; i++) {
+    paginated.push(items.slice(i * perPage - perPage, i * perPage));
+  }
+
+  return paginated;
+}
```

Comment:
The function 'getPaginatedItems' currently lacks a return type annotation. It's a good practice to explicitly specify the return type for better code readability and to avoid potential issues during compilation.

Reply:
fixed